### PR TITLE
[extended-monitoring] Add cert-exporter

### PR DESCRIPTION
**Pull request by kvaps**
## Description
This PR adds cert-exporter to the extended-monitoring module

## Why we need it and what problem does it solve?

cert-exporter allows to track certificates expiration.

fixes https://github.com/deckhouse/deckhouse/issues/365

## Changelog entries

```changes
module: extended-monitoring
type: feature
description: Add cert-exporter
note: |
  Added cert-exporter to track certificates expiration
```

### CODE REVIEW

File: ee/fe/modules/340-extended-monitoring/templates/cert-exporter/rbac-for-us.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{{- if .Values.extendedMonitoring.certificates.exporterEnabled }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cert-exporter
+  namespace: d8-monitoring
+  {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
```
Comment (nabokihms) [inline ee/fe/modules/340-extended-monitoring/templates/cert-exporter/rbac-for-us.yaml:8]:
```suggestion
{{- include "helm_lib_module_labels" (list .) | indent 2 }}
```
Just to keep the style. Or, we could switch to using `nindent` everywhere, but then there are a lot of changes to make.

File: ee/fe/modules/340-extended-monitoring/openapi/config-values.yaml [hunk 2]
```diff
@@ -52,6 +52,12 @@ properties:
         default: false
         description: |
           Enables certExporter.
+      namespaces:
```
Comment (nabokihms) [inline ee/fe/modules/340-extended-monitoring/openapi/config-values.yaml:55]: I have a proposal to stick to k8s-image-availability-exporter here:
1. Always collect metrics from all namespaces.
2. Alert about expiring certificates only if the extended monitoring enabled label present on the namespace.

Thus we could avoid adding the `namespaces` option to the config.

**Review by nabokihms — CHANGES_REQUESTED** [review]
Everything according to the code looks good, thank you. The only concern left is the namespaces option.

File: ee/fe/modules/340-extended-monitoring/openapi/config-values.yaml (hunk 2 above)
Comment (kvaps) [reply ee/fe/modules/340-extended-monitoring/openapi/config-values.yaml]: OK, solved in https://github.com/deckhouse/deckhouse/pull/479/commits/6402266d7eb82226ee1f84925321b277a965e095

File: ee/fe/modules/340-extended-monitoring/templates/cert-exporter/rbac-for-us.yaml (hunk 1 above)
Comment (kvaps) [reply ee/fe/modules/340-extended-monitoring/templates/cert-exporter/rbac-for-us.yaml:8]: I believe that using `nindent` will make the code more readable.
If you want to, I can prepare a PR to update it in other places.

Comment (kvaps) [reply ee/fe/modules/340-extended-monitoring/templates/cert-exporter/rbac-for-us.yaml:8]: solved in d9c4aa5b2dc24fa6f15b1ee5fb064c3ae6148dbc

**Review by nabokihms — APPROVED** [review]
Looks good, thank you.

**Pull request merged**